src/components/_alert.scss

```
@@ -7,104 +7,365 @@ category: Components
+    </h4>
+    <div>
+      詳しく見る
+      <i class="fa fa-angle-right ncgr-mar-l-4"></i>
+    </div>
+  </div>
+</a>
+<a class="ncgr-alert -color-success -appearance-flat ncgr-mar-b-8">
+  <div class="_leading">
+    <img class="_icon" src="./images/components/alert/alert-icon-white.svg">
+  </div>
+  <div class="_title">
+    <h4>
+      良好な状態
+    </h4>
+    <div>
+      詳しく見る
+      <i class="fa fa-angle-right ncgr-mar-l-4"></i>
+    </div>
+  </div>
+</a>
+<a class="ncgr-alert -color-warning -appearance-flat ncgr-mar-b-8">
+  <div class="_leading">
+    <img class="_icon" src="./images/components/alert/alert-icon-white.svg">
+  </div>
+  <div class="_title">
+    <h4>
+      注意喚起
+    </h4>
+    <div>
+      詳しく見る
+      <i class="fa fa-angle-right ncgr-mar-l-4"></i>
+    </div>
+  </div>
+</a>
+<a class="ncgr-alert -color-danger -appearance-flat">
+  <div class="_leading">
+    <img class="_icon" src="./images/components/alert/alert-icon-white.svg">
+  </div>
+  <div class="_title">
+    <h4>
+      発生中のエラー
+    </h4>
+    <div>
+      詳しく見る
+      <i class="fa fa-angle-right ncgr-mar-l-4"></i>
+    </div>
+  </div>
+</a>
 ```
 
 */
 
+$colors: default, primary, success, warning, danger;
+
 .ncgr-alert {
-  background-color: var(--color-default-50);
-  border-radius: 8px;
+  display: flex;
+  align-items: center;
+  border-radius: 0.5rem;
   box-sizing: border-box;
-  color: var(--color-default-900);
-  padding: 16px;
+  padding: 1rem;
   position: relative;
   width: 100%;
-  @media (prefers-color-scheme: dark) {
-    background-color: var(--color-default-200);
-  }
-  &.ncgr-alert--default {
-    background-color: var(--color-default-50);
-    color: var(--color-default-900);
-    @media (prefers-color-scheme: dark) {
-      background-color: var(--color-default-200);
+  @each $color in $colors {
+    &.-color-#{$color} {
+      color: var(--color-#{$color}-900);
+      &.-appearance-white {
+        background-color: var(--color-secondary-grouped-background);
+      }
+      &.-appearance-flat {
+        background-color: var(--color-#{$color}-50);
+        @media (prefers-color-scheme: dark) {
+          background-color: var(--color-#{$color}-200);
+        }
+      }
     }
   }
-  &.ncgr-alert--primary {
-    background-color: var(--color-primary-50);
-    color: var(--color-primary-900);
-    @media (prefers-color-scheme: dark) {
-      background-color: var(--color-primary-200);
-    }
+  ._leading {
+    display: flex;
+    padding-right: 1rem;
   }
-  &.ncgr-alert--success {
-    background-color: var(--color-success-50);
-    color: var(--color-success-900);
-    @media (prefers-color-scheme: dark) {
-      background-color: var(--color-success-200);
-    }
+  ._icon {
+    width: 3rem;
+    height: 3rem;
   }
-  &.ncgr-alert--warning {
-    background-color: var(--color-warning-50);
-    color: var(--color-warning-900);
-    @media (prefers-color-scheme: dark) {
-      background-color: var(--color-warning-200);
+  ._title {
```

@tsubasa-sris 

これtitleじゃなくてbodyのほうがいいかも
```suggestion
  ._body {
```